tsconfig.json

```diff
@@ -9,13 +9,14 @@
     "strict": true,
     "forceConsistentCasingInFileNames": true,
     "noFallthroughCasesInSwitch": true,
-    "module": "esnext",
+    "module": "commonjs",
```

Comment:
What we are doing with this is basically running CJS modules on the web (Its a nightmare). Use two separate build configs for frontend and Node.js. 
1. Create `tsconfig.base.json` - Use it as a common TS
2. Build Frontend app specific (react) - `tsconfig.frontend.json`
3. Build Node.js specific ones - `tsconfig.node.json`

Reply:
Thanks for pointing it out 👍 One caveat is that create-react-app doesn't give us an option to specify tsconfig file path but only takes `tsconfig.json` (related ticket: https://github.com/facebook/create-react-app/issues/6023) so I needed to use tsconfig.json for frontend.

Reply:
I did push the changes. I specified the tsc to compile using `tsconfig.node.json` which compile the `electron` directory. AFAIK the react code is compiled by its own process using `tsconfig.json` (I specified to include only `src` dir)